source/index.html.erb

```diff
@@ -2,52 +2,61 @@
 title: Jeff Stolz - Product Designer
 ---
 
-<h1><%= t("index.hello") %></h1>
-<h4><%= t("index.welcome") %></h3>
-<section class="index__cards">
-  <% link_to "mobile-heartbeat.html", class: "index__case" do %>
-    <div class="index__case--overlay">
-      <p class="index__case-description">
-        <%= t("index.mh-description") %>
-      </p>
-    </div>
-    <h5 class="index__case-heading">
-      <%= t("index.mh") %>
-    </h5>
-    <%= image_tag(
-      "mh-thumb.png",
-      class: "index__thumbnail",
-      alt: t("index.mh-alt"),
-    ) %>
-  <% end %>
-  <% link_to "hmh-marketplace.html", class: "index__case" do %>
-    <div class="index__case--overlay">
-      <p class="index__case-description">
-        <%= t("index.hmh-description") %>
-      </p>
-    </div>
-    <h5 class="index__case-heading">
-      <%= t("index.hmh") %>
-    </h5>
-    <%= image_tag(
-      "hmh-thumb.png",
-      class: "index__thumbnail",
-      alt: t("index.hmh-alt"),
-    ) %>
-  <% end %>
-  <% link_to "pegasystems.html", class: "index__case" do %>
-    <div class="index__case--overlay">
-      <p class="index__case-description">
-        <%= t("index.pega-description") %>
-      </p>
-    </div>
-    <h5 class="index__case-heading">
-      <%= t("index.pega") %>
-    </h5>
-    <%= image_tag(
-      "hmh-thumb.png",
-      class: "index__thumbnail",
-      alt: t("index.pega-alt"),
-    ) %>
-  <% end %>
-</section>
+<main class="index">
+
+  <h1><%= t("index.hello") %></h1>
+  <h4><%= t("index.welcome") %></h3>
+
+  <section class="index__cards">
+
+    <% link_to "mobile-heartbeat.html", class: "index__case" do %>
```

Comment:
I'd mark this as a thing to swing back and optimize. I think some of the duplication could be eliminated here. 

Not knowing how the Middleman works, you should be able to, say loop over a data set of index cards
```
for card in index-cards
 card.heading
 card.thumbnail
```
etc.

Reply:
Agreed - yeah I can probably loop through this easy enough with a Padrino helper